### DISCUSSION TITLE
how cdc writes data to target milvus?

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#37585] [Q&A and General discussion]
I have a question while using CDC. The target Milvus's `ttMsgEnabled` setting needs to be set to `false` for normal operation. I understand that when this setting is turned off, DML requests including insertions are not possible. How is CDC able to insert data into the target Milvus?!

@SimFG thank you for your works!
could check this discussion please? 🙇

**Comment by yhmo** [comment]
Wait @SimFG to comment.

**Comment by SimFG** [comment]
A new interface `ReplicateMessage` is used to forward all dmls uniformly
